init-mqtt.h

```diff
@@ -1,226 +1,241 @@
-#ifndef __INIT_MQTT__
-#define __INIT_MQTT__
-
-struct MqttConfig
-{
-  char server[MAX_STR_LEN];
-  int port;
-  char user[MAX_STR_LEN];
```

Comment:
please keep formatting proper
I cannot see any change because everth is reformatted